tests/unit/test_logger.py

```diff
@@ -196,7 +195,7 @@ def test_logger_init_bad_args():
 
 def test_logger_repr():
     logger = picologging.Logger("goo", picologging.DEBUG)
-    assert repr(logger) == "<Logger 'goo' (10)>"
+    assert repr(logger) == "<Logger 'goo' (DEBUG)>"
```

Comment:
Please add tests for the following: 

1. `__repr__` for invalid level no doesn't crash
2. `getLevelName()` for non integer returns TypeError
3. `getLevelName()` returns correct value for known level number

Reply:
Added tests for item 1 and 2 and item 3 was already done there in `test_picologging.py`.